### PR TITLE
Hyrule Compendium - First Version

### PR DESCRIPTION
**Pull request by HansZizold**
** DESCRIPTION **
The application shows the content of "The Legend of Zelda: Breathe of the Wild" game from the Hyrule Compendium API.
When the page loads, it shows the equipment items. There is a select element that allows change to other categories.
Based on the category different data is displayed at the bottom.
When selecting any item the application shows a new page with more detailed information about that item.
Testing routines are included.

Video Presenting the features of the application:

https://www.loom.com/share/a58dfe8b7c874440a9a2b09419f4ada9

** The name of the repo is musicartists because at first I chose another API, but had to change it.

### CODE REVIEW

**Comment by HansZizold** [comment]
hi, something funny happened with linters. I suppose is because of the publication on Netlify.
Anyway, in my local environment I don't have any linter error. Please consider.

image

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+
+## 🤝 Contributing
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
```
Comment (KDlamini) [inline README.md]: Please make sure that the "Acknowledgements" section or similar gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,68 @@
...
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (KDlamini) [inline README.md:73]: Please add a working link to your license or otherwise remove it.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @HansZizold 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating a working app by consuming your API of choice. 🎊 🥳 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  App works as expected
✔️  Redux used for state management
✔️  Good PR title description
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (HansZizold) [reply README.md]: Hi @KDlamini, ok you're right. I forgot this requirement.

**Comment by HansZizold** [comment]
@KDlamini, Thank you very much for your kindly review.
I think I made up for the observations. I am submitting again.

**Review by Deepakdanger — APPROVED** [review]
Hi @HansZizold, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**